docs/api.yml

```diff
@@ -533,3 +533,12 @@ components:
               userID:
                 type: string
                 example: "6620cf87-44ed-456e-9798-d5cbeb7fa10a"
+        images:
```

Comment:
We also need to update the model in _models/crowdaction.go_. We need a new struct similar to `CrowdactionParticipant` that is referenced by the struct `Crowdaction`.